src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx

```diff
@@ -41,37 +21,52 @@ const LocationPreferences = () => {
       </div>
       <div className={styles.right}>
         <div>
-          <TextInput
-            label="Country of residency"
-            placeholder="Start typing location"
-            value={country}
-            onChange={handleCountryChange}
-          />
+          <div className={errors.country ? styles.errorMsg : ''}>
+            <TextInput
+              label="Country of residency"
+              placeholder="Start typing location"
+              value={values.country}
+              onChange={handleChange}
+              name="country"
+            />
+            <p>{errors.country}</p>
+          </div>
+
           <CheckboxInput
             label="I’m open to residency relocation"
-            checked={openToRelocationCountry}
-            onChange={handleCheckboxChangeCountry}
+            checked={values.openToRelocationCountry}
+            onChange={handleChange}
+            name="openToRelocationCountry"
           />
         </div>
         <div>
-          <TextInput
-            label="City residency"
-            placeholder="Start typing location"
-            value={city}
-            onChange={handleCityChange}
-            addImportantIcon={true}
-          />
+          <div className={errors.city ? styles.errorMsg : ''}>
```

Comment:
**nitpick:** We were talking about making `InputWrapper` component for these, so it displays errors, and keep our inputs packed